src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const peopleMen = people.filter(item => item.sex === 'm')
+    .filter(item => century
+      ? Math.ceil(item.died / 100) === century : true);
```

Comment:
```suggestion
  const peopleMen = people.filter(item => century
      ? Math.ceil(item.died / 100) === century &&  item.sex === 'm' :  item.sex === 'm');
```
redundant two itereations. combine into one
check similar cases in other functions